packages/page-accounts/src/Sidebar/Sidebar.spec.tsx

```diff
@@ -0,0 +1,128 @@
+// Copyright 2017-2021 @polkadot/page-accounts authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import { fireEvent, screen, within } from '@testing-library/react';
+
+import { MemoryStore } from '@polkadot/test-support/keyring';
+import { keyring } from '@polkadot/ui-keyring';
+
+import { AccountOverrides } from '../../test/hooks/default';
+import { AccountsPage } from '../../test/pages/accountsPage';
+
+const anAccount = (): AccountOverrides => ({});
+const aliceAddress = '5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY';
+const bobAddress = '5FHneW46xGXgs5mUiveU4sbTyGBzmstUspZC92UhjJM694ty';
+
+describe('Sidebar, subs list', () => {
+  let accountsPage: AccountsPage;
+
+  beforeAll(() => {
+    if (keyring.getAccounts().length === 0) {
+      keyring.loadAll({ isDevelopment: true, store: new MemoryStore() });
+    }
+  });
+
+  beforeEach(() => {
+    [aliceAddress, bobAddress].forEach((address) => keyring.forgetAccount(address));
+    accountsPage = new AccountsPage();
+  });
+
+  it('dose not display subs when account has zero subs', async () => {
+    accountsPage.renderPage([[aliceAddress, anAccount()]]);
+
+    const accountRows = await accountsPage.findAccountRows();
+
+    fireEvent.click(await within(accountRows[0].primaryRow).findByTestId('account-name'));
+
+    const sidebar = await screen.findByTestId('sidebar');
+    const identitySection = await within(sidebar).findByTestId('identity-section');
+
+    const subs = within(identitySection).queryAllByText('sub');
+
+    expect(subs).toHaveLength(0);
+  });
+
+  it('displays amount of subs', async () => {
```

Comment:
```suggestion
  it('displays count of subs', async () => {
```
Count or number, amount goes better with money or uncountable nouns.

Reply:
✅